ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -24,8 +30,8 @@ time, mark, audio, video {
   padding: 0;
   border: 0;
   font-size: 100%;
-  font: inherit;
-  font-family: $kuiFontFamily;
+  font: inherit; /* 1 */
+  font-family: inherit; /* 1 */
```

Comment:
@snide Here's a good example of one of the benefits of following the [style guide's commenting recommendation](https://cssguidelin.es/#low-level). The comment applies to multiple properties, and we can use the citation number to communicate that.